app/components/autocomplete/autocomplete.tsx

```diff
@@ -192,14 +197,18 @@ const Autocomplete = ({
                         hasFilesAttached={hasFilesAttached}
                     />
                     }
-                    {/* <SlashSuggestion
+                    {showCommands &&
+                    <SlashSuggestion
```

Comment:
Unmounting this component like this breaks the command cache. We can store the command cache somewhere else and not in the component, but then we have to handle how long do we trust the cache, etc... (it was previously done in redux). 0/5 whether we should worry about this, since this is for uncommon cases.

Reply:
In memory singleton were you ask for the commands for certain team, if the cache is "fresh" get it from the cache, if not, it ask the server. It seems fairly simple. Will do.

Regarding how uncommon, now that I think about it, may be more common than I though. Any moment that while you are writing a command you use at-mention, channel-mentions, or emojis, it will unmount the component, and then has to mount it again and fetch again.